app/models/queued_email.rb

```diff
@@ -223,30 +229,24 @@ def send_email
     return true unless RunLevel.is_normal?
 
     log_msg = "SEND #{flavor} " \
-      "from=#{begin
-                user.login
-              rescue StandardError
-                "nil"
-              end} " \
-      "to=#{begin
-              to_user.login
-            rescue StandardError
-              "nil"
-            end} " +
-              queued_email_integers.map { |x| "#{x.key}=#{x.value}" }.
-              join(" ") +
-              queued_email_strings.map { |x| "#{x.key}=\"#{x.value}\"" }.
-              join(" ")
+      "from=#{user&.login || "nil"} " \
+      "to=#{to_user&.login || "nil"} " +
+              queued_email_integers.map { |x|
+                "#{x.key}=#{x.value}"
+              }.join(" ") +
+              queued_email_strings.map do |x|
+                "#{x.key}=\"#{x.value}\""
+              end.join(" ")
     self.class.debug_log(log_msg)
     current_locale = I18n.locale
-    result = false
-    if user == to_user
-      unless Rails.env.test?
-        raise("Skipping email with same sender and recipient: #{user.email}\n")
-      end
-    else
-      result = deliver_email
-    end
+    # result = false
+    # if user == to_user
+    #   unless Rails.env.test?
+    #     raise("Skipping email with same sender and recipient: #{user.email}\n")
+    #   end
+    # else
+    result = deliver_email
+    # end
```

Comment:
...and here's the deletion of the check if `user == to_user`. Delivers mail regardless.

Reply:
Actually, I was thinking more along the lines of checking `if user.present? && user == to_user`.  It continues not to make sense to send notifications to a user that they changed something themselves! :)  But if both user and to_user are nil, then it is probably this admin message, and *then* we should send it even though user == to_user.  Does that make sense?